docker/agent/Dockerfile.agent

```diff
@@ -0,0 +1,38 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG BASE_IMAGE='adoptopenjdk/openjdk8:alpine'
+
+FROM alpine as build
+
+ARG DIST_NAME
+
+ADD "$DIST_NAME.tar.gz" /
+
+RUN mv /$DIST_NAME /skywalking
+
+FROM apache/skywalking-base:8.4.0-es6 AS cli
+
+FROM $BASE_IMAGE
+
+LABEL maintainer="kezhenxu94@apache.org"
+
+ENV JAVA_TOOL_OPTIONS=-javaagent:/skywalking/agent/skywalking-agent.jar
+
+WORKDIR /skywalking
+
+COPY --from=build /skywalking/agent /skywalking/agent
+COPY --from=cli /skywalking/bin/swctl /skywalking/bin
```

Comment:
Why we need swctl?

Reply:
> Why we need swctl?

I was thinking that users might need `swctl` to troubleshoot, for example, when the agent failed to connect to the backend, or use `swctl` to report other events than `start / shutdown` event.

Reply:
This case makes sense to me. But from the docker file perspective, why it comes from an older skywalking release, rather than CLI release directly from ASF?